docs/index.md

```diff
@@ -33,10 +34,13 @@ instructions for the Docker-based deployment as it is more portable and cross-pl
 1. First download the docker-compose file from [here](./files/docker-compose.yaml).
 
 2. On a terminal opened in the same directory where you download the docker-compose.yaml file you
-can simply type: `docker compose up -d grafana`
+can simply type:
+
+        docker compose up -d grafana
+
```

Comment:
this must be wrapped in a code block

```suggestion

```console 
docker compose up -d grafana
```

```

Reply:
It is a code block.

Reply:
not a fenced code block, not syntax highlight either